squeal-postgresql/src/Squeal/PostgreSQL/Type/Schema.hs

```diff
@@ -440,6 +441,22 @@ type family SetSchema sch0 sch1 schema0 schema1 obj srt ty db where
     (Create obj (srt ty) schema1)
     (Alter sch0 (DropSchemum obj srt schema0) db)
 
+{- | `SubDB` checks that one `SchemasType` is a sublist of another,
+with the same ordering.
```

Comment:
Why do we care about the ordering?

Reply:
We don't necessarily. It might be worth writing the additional slower type families that check up to re-ordering.

Reply:
I think its interesting, cause we sort of do in the case of duplicates.